Week06/lesson12/homework/rock_paper_scissors_starter/js/main.js

```diff
@@ -0,0 +1,27 @@
+// storing the running totals, starting at 0
+var humanScore = 0;
+var computerScore = 0;
+
+// function human clicks rock, scores dont change
+function rock() {
+  $("rock").click;
+  $("status").innerHTML = "The bot played Rock, you draw!";
+  $("humanScore").innerHTML = humanScore;
+  $("computerScore").innerHTML = computerScore;
+}
+
+// function human clicks paper, add one to human score
+function paper() {
+  $("paper").click;
+  $("status").innerHTML = "The bot played Rock, you win!";
```

Comment:
If you're going to use jQuery to grab the element, remember to use a jQuery method, like so:

`$("#status).html("The bot played Rock, you win!");`

... `innerHTML` won't work on a jQuery wrapped method